vpr/src/place/place.cpp

```diff
@@ -2077,7 +2167,8 @@ static e_move_result assess_swap(double delta_c, double t) {
 
     float fnum = vtr::frand();
     float prob_fac = std::exp(-delta_c / t);
-    if (prob_fac > fnum) {
+    float prob_fac_cong = std::exp(-cong_delta_c / t);
+    if (prob_fac > fnum && (prob_fac_cong>fnum || placer_opts.place_algorithm != CONGESTION_AWARE_PLACE)) {
```

Comment:
I don't think we need to calculate exp(-cong_delta_c/t) as an individual term in here. The delta_c already has that term